src/PantheonTerminalWindow.vala

```diff
@@ -49,39 +49,69 @@ namespace PantheonTerminal {
             }
         """;
 
-        const string ui_string = """
-            <ui>
-            <popup name="MenuItemTool">
-                <menuitem name="New window" action="New window"/>
-                <menuitem name="New tab" action="New tab"/>
-                <menuitem name="CloseTab" action="CloseTab"/>
-                <menuitem name="Copy" action="Copy"/>
-                <menuitem name="Paste" action="Paste"/>
-                <menuitem name="Select All" action="Select All"/>
-                <menuitem name="Search" action="Search"/>
-                <menuitem name="About" action="About"/>
-
-                <menuitem name="NextTab" action="NextTab"/>
-                <menuitem name="PreviousTab" action="PreviousTab"/>
-
-                <menuitem name="ZoomIn" action="ZoomIn"/>
-                <menuitem name="ZoomOut" action="ZoomOut"/>
-
-                <menuitem name="Fullscreen" action="Fullscreen"/>
-            </popup>
-
-            <popup name="AppMenu">
-                <menuitem name="Copy" action="Copy"/>
-                <menuitem name="Paste" action="Paste"/>
-                <menuitem name="Select All" action="Select All"/>
-                <menuitem name="Search" action="Search"/>
-                <menuitem name="Show in File Browser" action="Show in File Browser"/>
-            </popup>
-            </ui>
-        """;
+        public SimpleActionGroup actions { get; construct; }
+
+        public const string ACTION_PREFIX = "win.";
```

Comment:
Do we really need this constant here? We can just add the `win.` prefix to all those actions and do not care about adding this prefix everytime we use an action.